app/ts/Subproblem/ScaleRanges/ScaleRangesUtil.test.ts

```diff
@@ -138,4 +139,24 @@ describe('ScaleRangesUtil', () => {
       expect(result).toBeFalsy();
     });
   });
+
+  describe('hasRowWithOnlySameValue', () => {
+    it('should return true if there is a row with equal lower and upper bounds', () => {
+      const observedRanges: Record<string, [number, number]> = {dsId: [0, 0]};
```

Comment:
I feel like testing a function that operates over a collection only with a single-element collection isn't quite covering the bases